### ISSUE TITLE
[SOLVED] "included folders" option removed (blocks access to folders with permissions!)

### ISSUE THREAD

**Issue by lightful** [SimpleMobileTools/Simple-Gallery#2499]
The latest versions do not offer the "included folders" option (or at least I have not been able to find it). So it is not possible to see photos in paths where Android still allows access (an example bellow).

I'm using Android 11 and the latest version of this app from F-Droid (targeting API 31) correctly shows the included folders I configured under previous versions (e.g. /storage/emulated/0/photos) and I can see ALL my photos there. So indeed IT HAS PERMISSION... but now I can't add more folders like this one.

So this is not an Android-related problem, I feel that the option to manually add folders (at worst by full path text input) should still be there for power users. In addition, now it doesn't shows the screenshots (stored in a folder as accessible as the previously mentioned one).

Meanwhile I have resorted to manually downgrade to 6.21.2.

**Comment by tibbi** [comment]
file fetching had to be updated on Google Play, the app no longer has access to All files. Only the f-droid variant has it. So included folders cannot work anymore either.

**Comment by lightful** [comment]
I was using the version from F-Droid (I think) and it missed the included folders option. I say "I think" because despite I installed it from F-Droid, F-Droid it was misbehaving (the app didn't appear on F-Droid apps list -> only when I searched for it... but neither in Play Store list of apps). Finally I uninstalled it (instead of upgrading/downgrading) and now the latest version properly shows the "included folders" option. This even if I restore the settings exported in the past!.

So it seems a problem related with settings upgrading... or perhaps I initially installed this app from Play Store and later updated it from F-Droid?. Anyway, a clean install from F-Droid has solved the problem.

I apologize for the false report.
